Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,62 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Runtime.ExceptionServices;
+using System;
+using System.Reflection.Metadata.Ecma335;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
+        protected Item[] _items;
+
+        // displays all items in inventory
+        // renamed from 'GetItems' to show the clear different from the 'GetItems' method below
+        public void ShowAllItems()
         {
-            return new Item[0];
+            if (!Array.TrueForAll(_items, IsNull => IsNull == null))
```

Comment:
What if array is empty?

Reply:
Try to fail early and avoid negations.
If there are no elements- end the function.
if(Array.TrueForAll(_items, IsNull => IsNull == null))
{
  Console.WriteLine("The inventory is currently empty.");
  return;
}
// no need else. Reduced nesting.